### DISCUSSION TITLE
How to change the content of the widgets on the FilamentPHP dashboard page?

### DISCUSSION THREAD

**Discussion by enessvg** [filamentphp/filament#15100] [Help] [answered]
### Package

Panel builder

### Package Version

3.2

### How can we help you?

<img width="1403" alt="Ekran Resmi 2024-12-16 10 53 39" src="https://github.com/user-attachments/assets/2c897df3-22cd-4772-a740-b57b4908416f" />

Hi, as I said in the title, where else can I change the places that appear in the picture, not from the “vendor” folder?

**Comment by leandrocfe** [comment]
They are in your panel provider

```php
$panel
...
->widgets([
    Widgets\AccountWidget::class,
    Widgets\FilamentInfoWidget::class,
])
```

You can hide them and create [custom widgets](https://filamentphp.com/docs/3.x/panels/dashboard#available-widgets)

**Reply by enessvg** [reply]
Can we change their content?

**Reply by leandrocfe** [reply, marked as answer]
Create a custom widget

```bash
php artisan make:filament-widget CustomInfoWidget
```

You can copy and paste [the original content](https://github.com/filamentphp/filament/blob/3.x/packages/panels/resources/views/widgets/filament-info-widget.blade.php) into your view file `resources/views/filament/widgets/custom-info-widget.blade.php`, then edit it as needed. If you are using Tailwind classes that Filament doesn't include, create a [custom theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme).
